libs/sdk-ui-tests-e2e/cypress/tools/dashboards.ts

```diff
@@ -157,6 +176,16 @@ export class TopBar {
         this.clickShareButton();
         return this;
     }
+
+    deleteDashboard(confirm: boolean) {
```

Comment:
should be written in dashboardHeader.ts

Reply:
I think it belongs in TopBar() in dashboards.ts is reasonable.

Reply:
dashboardHeader.ts and TopBar are duplicated. Should move this method to dashboardHeader
then create a follow-up ticket to move all functions in TopBar to dashboardHeader.ts